README.md

```diff
@@ -36,16 +36,16 @@ Read the [Changelog](https://github.com/Microsoft/vscode-go/blob/master/CHANGELO
 
 ### IntelliSense
 
-- Auto Completion of symbols as you type (using `gocode`)
-- Signature Help for functions as you type (using `gogetdoc` or `godef`+`go doc`)
-- Quick Info on the symbol as you hover over it (using `gogetdoc` or `godef`+`go doc`)
+- Auto Completion of symbols as you type (using `gopls` or `gocode`)
```

Comment:
How about the below instead?

> using language server or `gocode`

Otherwise, I worry that the message is that `gopls` is just another tool

Reply:
Done.